src/YieldDisburser.sol

```diff
@@ -20,6 +20,14 @@ contract YieldDisburser is OwnableUpgradeable {
 
     event BaseYieldDistributed(uint256 amount, address project);
 
+    error YieldNotResolved();
+    error YieldTooLow(uint256);
+    error EndAfterCurrentBlock();
+    error IncorrectNumberOfProjects();
+    error NoCheckpointsForAccount();
+    error StartMustBeBeforeEnd();
+    error MinimumTimeBetweenClaimsMustBeGreaterThanZero();
```

Comment:
this one is a bit longwinded. "MustBeGreaterThanZero()" is probably fine and can be used in multiple places

Reply:
amended by ba14af2